### PR TITLE
PAY-1346 - added PayTo to CustomerApi, added tests for SetDefault and…

### PR DESCRIPTION
**Pull request by volodymyr32**
… update customer with DefaultPaymentMethodId

### CODE REVIEW

File: src/main/java/com/payfurl/payfurlsdk/api/CustomerApi.java [hunk 1]
```diff
@@ -5,16 +5,7 @@
 import com.payfurl.payfurlsdk.auth.AuthHandler;
 import com.payfurl.payfurlsdk.auth.AuthType;
 import com.payfurl.payfurlsdk.http.client.HttpClient;
-import com.payfurl.payfurlsdk.models.CustomerData;
-import com.payfurl.payfurlsdk.models.CustomerList;
```
Comment (julia-software) [inline src/main/java/com/payfurl/payfurlsdk/api/CustomerApi.java:9]: Please do not join imports, in JB products you could switch off:
https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports 

https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

File: src/main/java/com/payfurl/payfurlsdk/models/PayIdDetails.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        this.payIdType = payIdType;
+    }
+
+    public String getPayId() {
+        return payId;
+    }
+    public String getPayIdType() {
+        return payIdType;
+    }
+
```
Comment (julia-software) [inline src/main/java/com/payfurl/payfurlsdk/models/PayIdDetails.java:25]: Please remove empty line

File: src/main/java/com/payfurl/payfurlsdk/models/PayIdDetails.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        public Builder withPayId(String payId) {
+            this.payId = payId;
+            return this;
+        }
+
+        public Builder withPayIdType(String payIdType) {
+            this.payIdType = payIdType;
+            return this;
+        }
+
```
Comment (julia-software) [inline src/main/java/com/payfurl/payfurlsdk/models/PayIdDetails.java:41]: Please remove empty line

Comment (julia-software) [reply src/main/java/com/payfurl/payfurlsdk/models/PayIdDetails.java:41]: Also, you could use formatting that we usually use 
image

**Review by julia-software — CHANGES_REQUESTED** [review]

File: src/main/java/com/payfurl/payfurlsdk/api/CustomerApi.java (hunk 1 above)
Comment (volodymyr32) [reply src/main/java/com/payfurl/payfurlsdk/api/CustomerApi.java]: Fixed.

**Pull request merged**